src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,11 +2,15 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Method 'login' was called. Params: login=" + login);
```

Comment:
Try to put parameters into message using specific syntax of logger: {}
```suggestion
        logger.info("Method 'login' was called. Params: login={}",  login);
```